### PR TITLE
Fix send.test.js timing and config errors

### PR DESCRIPTION
**Pull request by dorianvp**
**Problems:** 
- Detox tests are failing to interact with bottom buttons due to a specific warning dialog that appears when the Reduced Motion setting is enabled. This impacts the test flow, as detailed [here](https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshooting/#reduced-motion-setting-is-enabled-on-this-device)
- The `send.test.js` file has timing issues because the `testID` was not updating in response to state changes on a button, leading to inconsistent test behavior.

**This PR fixes both issues by ensuring:**
- Proper handling of the warning dialog to prevent interference with the button taps.
- The `Send` component correctly updates the `testID` to reflect state changes, ensuring accurate test behavior.

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { ModeEnum, RouteEnums } from './app/AppState';
 import { LogBox } from 'react-native';
 
 LogBox.ignoreLogs(['Require cycle: app\\uris\\serverUris.mock.ts -> app\\uris\\serverUris.mock.ts']);
+LogBox.ignoreLogs(['[Reanimated] Reduced motion setting is enabled on this device.']);
```
Comment (juanky201271) [inline App.tsx]: since `ignoreLogs` accepts an array, maybe is better to put both warnings together...

File: components/Send/Send.tsx [hunk 2]
```diff
@@ -1651,7 +1651,7 @@ const Send: React.FunctionComponent<SendProps> = ({
                 marginBottom: 20,
               }}>
               <Button
-                testID="send.button"
+                testID={sendButtonEnabled ? 'send.button' : 'send.button-disabled'}
```
Comment (juanky201271) [inline components/Send/Send.tsx:1654]: good point.

**Review by juanky201271 — CHANGES_REQUESTED** [review]
a minor change...

File: App.tsx (hunk 1 above)
Comment (dorianvp) [reply App.tsx]: Thanks for pointing that out! I'll go ahead and update the PR accordingly.

**Review by juanky201271 — APPROVED** [review]
nice job!!!!

**Pull request merged**